app/src/main/java/com/example/myapplication/UserModel.java

```diff
@@ -37,4 +50,21 @@ void doAction() throws IOException {
         osw.close();
         fos.close();
     }
+
+    public static void createFile(String fileName){
```

Comment:
This doesn't seem to be related to the transfer issue - doesn't need to be in this PR

Reply:
this was when I was working on the file write - since it's on the same page as changes related to this PR, how would I not add it to this one? Do I have to make separate branches ahead of time?

Reply:
Once you find out that you are going to split your work in two parts (part 1: user model / match data; part 2: files), you can create a new branch from your existing branch (e.g., CamilaFileSave). In that new branch, this code would remain, while on the other branch you can delete it.